### PR TITLE
Web project content

### PR DESCRIPTION
**Pull request by ommarz**
Add index.html file
Add stylesheet.css
Add README.md

### CODE REVIEW

File: README.md
Comment (ommarz) [inline README.md:1]: README.md is usually added to show information on your project. Kindly make changes so it is related to your project. Remember you can always use [this template](https://github.com/microverseinc/readme-template) but you need to customize it for your project

File: stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        115deg,
+        rgba(58, 58, 158, 0.8),
+        rgba(136, 136, 206, 0.7)
+      ),
+      url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+}
+form{
+    width: 640px;
+    background-color: rgb(39, 38, 69);
+    color:white;
```
Comment (ommarz) [inline stylesheet.css:14]: I suggest leaving spaces between semicolons and styling.

File: stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    /*text-align: center;*/
+    color:white;
+        background-image: linear-gradient(
+        115deg,
+        rgba(58, 58, 158, 0.8),
+        rgba(136, 136, 206, 0.7)
+      ),
+      url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+}
+form{
```
Comment (ommarz) [inline stylesheet.css:11]: I suggest making sure there is a space between classes and curly brackets. 
```suggestion
form {
```

**Review by ommarz — COMMENTED** [review]
### Almost There!!!

- [ ] Please make sure that [linters](https://github.com/microverseinc/linters-config) are installed.

Comments marked as _*OPTIONAL_ are not required to be completed.

File: README.md
Comment (stanleySimeon) [reply README.md:1]: Great work so far, but I suggest you make sure your comments are descriptive as much as possible. 😄

File: stylesheet.css (hunk 2 above)
Comment (stanleySimeon) [reply stylesheet.css:11]: As I told you before, kindly make sure you explain well why you requested the changes and please, be more polite in your comments. 😄

File: stylesheet.css (hunk 1 above)
Comment (stanleySimeon) [reply stylesheet.css:14]: As mentioned above, kindly be more specific, more explicit and make sure your comment is descriptive and polite! 😄

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @ommarz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (ommarz) [reply README.md:1]: You're doing well adding a README.md.  Remember that this type of document is usually added to show information about your project. You can always use [this template](https://github.com/microverseinc/readme-template) just remember to customize it according to your project.

File: stylesheet.css (hunk 1 above)
Comment (ommarz) [reply stylesheet.css:14]: Good job with your styling! Also, remember that to make your code easier to read, you need to leave spaces between semicolons and styling, as follows:
```suggestion
    color:  white;
```

File: stylesheet.css (hunk 2 above)
Comment (ommarz) [reply stylesheet.css:11]: Good job with your styling! Also, remember that to make your code easier to read, you need to leave spaces between semicolons and styling, as follows:
```suggestion
form {
```

**Review by ommarz — COMMENTED** [review]
**_Required Changes_** ♻️
_Check the comments under the review._

_Optional suggestions_
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @ommarz, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Comments are descriptive ✅
- Comments are clear and polite ✅


## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**